pkg/apis/core/validation/shoot_test.go

```diff
@@ -405,6 +405,14 @@ var _ = Describe("Shoot Validation Tests", func() {
 					})),
 				))
 			})
+			It("should allow to set annotation where none existed", func() {
```

Comment:
```suggestion
			It("should allow to set the HAControlPlanes options if it did not exist before", func() {
```
To make it more aligned with the description of the tests above.